jupyter_alabaster_theme/static/pcss/mediaqueries.pcss

```diff
@@ -158,6 +158,13 @@
       display: none;
     }
 }
+/* Medium sized devices */
+@media (max-width: 768px) {
```

Comment:
Just moved and committed the changes to group the media queries but when I run `npm run css` it returns this:

![screen shot 2017-05-26 at 10 31 55 am](https://cloud.githubusercontent.com/assets/6437976/26505683/929eb32e-41fe-11e7-89f3-96e59a8b09da.png)